entity-framework/core/what-is-new/roadmap.md

```diff
@@ -11,9 +11,21 @@ uid: core/what-is-new/roadmap
 > [!IMPORTANT]
 > Please note that the feature sets and schedules of future releases are always subject to change, and although we will try to keep this page up to date, it may not reflect our latest plans at all times.
 
+## Last release: EF Core 2.1
+
 The stable version of EF Core 2.1 was released on May 30, 2018. You can find more information about this release in [What is new in EF Core 2.1](xref:core/what-is-new/ef-core-2.1).
 
-We have not completed the [release planning process](#release-planning-process) for the next release after 2.1.
+## Future releases
+
+### EF Core 2.2
+
+This release will include many bug fixes, and relatively small number of new features. Details about this release are included in the the [EF Core 2.2 roadmap announcement](https://github.com/aspnet/Announcements/issues/308). 
```

Comment:
I am torn between linking to the announcement and moving the description here are updating the announcement to point here.

Of course I could just duplicate the contents.

Reply:
I don't have a strong preference either way--I can see advantages of both.